cmd/xdpcap/flags.go

```diff
@@ -5,9 +5,13 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"regexp"
+	"strconv"
 	"strings"
 
+	"github.com/google/gopacket/layers"
```

Comment:
Is this already a dependency? If not, I think it's worth avoiding adding this. gopacket is huge.

Reply:
Maybe some of the constants from https://godoc.org/golang.org/x/sys/unix would do?

Reply:
Unfortunately we already have to pull in `gopacket/layers` to use it with `gopacket/pcap` (the `libpcap` interface).